enterprise/frontend/src/embedding-sdk/cli/utils/ask-for-db-connection-info.ts

```diff
@@ -33,19 +33,6 @@ export async function askForDatabaseConnectionInfo(options: Options) {
       continue;
     }
 
-    const visibleIf = field["visible-if"];
```

Comment:
this was my [original comment](https://github.com/metabase/metabase/pull/46810#discussion_r1718046281).

Basically, I don't believe this ever works as we expected.